src/types/Person.ts

```diff
@@ -7,3 +7,7 @@ export interface Person {
   motherName: string | null,
   slug: string,
 }
+
+export type SetAutoComplList = (p: Person[], query: string) => Person[];
```

Comment:
Avoid one-letter names for variables/parameters. And as I said previously you actually don't need to create types for functions